README.md

```diff
@@ -50,13 +50,13 @@ Maven:
 <dependency>
     <groupId>uk.gov.hmcts.reform</groupId>
     <artifactId>java-logging-httpcomponents</artifactId>
-    <version>1.2.1</version>
+    <version>1.2.2</version>
 </dependency>
 ```
 
 Gradle:
 ```groovy
-compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: '1.2.1'
+compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: '1.2.2'
```

Comment:
I probably should've consulted somebody about versioning. Do we update versions in readme after the release?

Reply:
we update in the PR that is planned to be released, i.e. if you're planning on releasing after this change then update now.

otherwise do a separate PR when you're ready with the version changes